src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1630,6 +1654,72 @@ private void MakeSeparatorList(ReadOnlySpan<char> separators, ref ValueListBuild
             }
         }
 
+        private void MakeSeparatorListVectorized(ref ValueListBuilder<int> sepListBuilder, char c, char? c2 = null, char? c3 = null)
+        {
+            // Constant that defines indices of characters within an AVX-Register
+            const ulong indicesConstant = 0xFEDCBA9876543210;
+            ReadOnlySpan<byte> shuffleConstantData = new byte[] {
```

Comment:
Are future readers going to be able to figure out where shuffleConstantData comes from (is there a comment elsewhere maybe?)

Reply:
Added a comment in https://github.com/dotnet/runtime/pull/38001/commits/65cba0ca5ccc2bb55fa8044cb7326e2e0b6a455a